### ISSUE TITLE
Inconsistent Caps Lock behavior

### ISSUE THREAD

**Issue by zasma** [swaywm/sway#1266]
I started to have a strange problem with Caps Lock last winter on two Arch Linux machines at the same time (after a system update). Caps Lock functions as Shift now and is only activating as Caps Lock when using the modifiers Alt or Shift, even though no `setxkbmap` option is set. This problem only occurs in `sway`, not in the shell nor in `weston`.

Using `setxkbmap -query` gives:
```
rules:      evdev
model:      pc105
layout:     us
```

I identified a solution this week, but I cannot figure out why it works and why it only works under specific conditions. Using `setxkbmap -option caps:none` or `setxkbmap -option caps:shiftlock` after starting `sway` enables the normal Caps Lock behavior with:
```
rules:      evdev
model:      pc105
layout:     us
options:    caps:none
```

I tried to implement this solution by using a script to start `sway` with
```
#!/bin/bash
export XKB_DEFAULT_OPTIONS=caps:none;
sway -d 2> ~/.sway.log;
```

This completely deactivates the Caps Lock key though, as it should. Did anyone else experience/solve this issue?

**Comment by kaberc** [comment]
I have simular issue
```
export XKB_DEFAULT_LAYOUT=us,ru
export XKB_DEFAULT_VARIANT=,winkeys
export XKB_DEFAULT_OPTIONS=grp:caps_toggle
```
After that caps doesn't toggle layouts. It toggles only after I press shift or alt keys. The same issue with rctrl_toggle and shifts_toggle. Maybe it's a different issue, I'll post it then.

**Comment by Hummer12007** [comment]
Same for me: caps lock has any effect only after another modifier key is pressed.

**Comment by Zer0-One** [comment]
Same issue ^

```
zero-one@electron ~ $ setxkbmap -verbose -query
Trying to build keymap using the following components:
keycodes:   evdev+aliases(qwerty)
types:      complete
compat:     complete
symbols:    pc+us+inet(evdev)
geometry:   pc(pc105)
rules:      evdev
model:      pc105
layout:     us
```

**Comment by SirCmpwn** [comment]
"me too" ad nauseam isn't helpful. "Here's a patch" is helpful.

**Comment by sameer** [comment]
Looks like KDE had a similar issue and fixed it: https://bugs.kde.org/show_bug.cgi?id=377155

I've been investigating for a bit but I can't tell if it's the same.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
